packages/core/src/Page/PageVueServerRenderer.ts

```diff
@@ -111,6 +123,6 @@ const pageVueServerRenderer = {
   pageEntries,
 };
 
-module.exports = {
+export = {
```

Comment:
Since this file only uses named exports, this could be a good place to use es6 syntax instead:

```
export async function updateMarkBindVueBundle(newBundle : string) : Promise<void>
```

Reply:
I think doing 
```
export const pageVueServerRenderer = { ... }
```
would be better than exporting the individual functions since it would be more obvious to the reader when using functions from `PageVueServerRenderer`. Wydt?

Reply:
sounds good, I misread the original code as `module.exports = pageVueServerRenderer`. 🤦